_redirects.yml

```diff
@@ -563,6 +563,8 @@
 # feedback
 "https://docs.google.com/forms/d/e/1FAIpQLSenjK8KiBiOrwCpq06u_iVEZGv4tfTEpfmY_cQDmsqJrHL9Gw/viewform":
   - /feedback/extension/
+"https://docs.google.com/forms/u/1/d/e/1FAIpQLScgzKwqQyr7BmnphdrcGTO3wkye_McHJVsWzOIgEESfqmditQ/formResponse":
```

Comment:
Interesting; what's the difference between `viewForm` and `formResponse` ?

Reply:
`formResponse` is the path where form response payloads get `POST`ed to, which is what we're interested in for this usecase (feedback plugin submitting the response programatically). `GET`ing it redirects to `viewForm` though, so if we intend to use this URL in other scenarios (such as linking from DD/docs), it'll work there as well.

Reply:
Oh! I thought this was the link that we used in Docker Desktop (which ideally wouldn't be hard-coded to a form either). Wondering if we can find some way to use the same redirect, but guess that's not an option 🤔

Reply:
That's what I meant – we can use the same one, but the one we define as the redirect should be `formResponse` since we can `GET` that and it will redirect to the correct page (`viewForm`), but for `POST`ing from the plugin it needs to be `formResponse`.